modules/ROOT/pages/add-tags.adoc

```diff
@@ -1,14 +1,18 @@
 = Identifying APIs to Govern
 
-APIs that meet the filter criteria in API Governance profiles become _governed APIs_. Identifying the APIs to govern before creating your governance profiles helps ensure that you are governing the APIs in the way you want to govern them and that conformance notifications are sent only to the API owners who are ready to begin making their APIs conformant to your governance profiles.
+APIs that meet the filter criteria in API Governance profiles become _governed APIs_. 
+Identify the APIs to govern before creating your governance profiles to help ensure that:
+
+* You are governing only the APIs that you want to govern
+* API owners are ready for governance before they start receiving conformance notifications
 
 == Add Tags and Categories to APIs in Exchange
 
 To identify sets of APIs to govern, apply tags and categories to the API assets in Exchange. You can then select tags and categories as filter criteria when you create your governance profiles. 
 
-* To identify the APIs to govern by adding asset tags to the API assets in Exchange, see xref:exchange::to-describe-an-asset.adoc#add-and-remove-asset-tags[Add and Remove Asset Tags in Exchange].
+* To add asset tags, see xref:exchange::to-describe-an-asset.adoc#add-and-remove-asset-tags[Add and Remove Asset Tags in Exchange].
 
-* To identify the APIs to govern by adding categories to the API assets in Exchange, see xref:exchange::to-describe-an-asset#add-and-remove-categories.adoc[Add and Remove Categories in Exchange].
+* To add categories, see xref:exchange::to-describe-an-asset#add-and-remove-categories.adoc[Add and Remove Categories in Exchange].
```

Comment:
Does this link resolve?

Reply:
It does as of now. Do you know of some changes pending in the Exchange doc that might affect this?

Reply:
Does the link go to where you intend? I ask because the .adoc is in a strange place.

Reply:
Yes, I see what you mean. It does work in production and I didn't change it, but it probably shouldn't work. Probably some trick Gary did in the Antora build. I'll fix it.

Reply:
In prod it resolves properly: 
https://docs.mulesoft.com/api-governance/add-tags#add-tags-and-categories-to-apis-in-exchange

Reply:
Fixed

Reply:
It goes to this [topic](https://docs.mulesoft.com/exchange/to-describe-an-asset#add-and-remove-categories.adoc), but I think you want it to go to this [one](https://docs.mulesoft.com/exchange/to-describe-an-asset#add-and-remove-categories). But I could be wrong.